test/library/packages/TOML/BurntSushi/TomlTest.future

```diff
@@ -1,4 +1,8 @@
 feature request: Improve TOML library
 Issue #7104
 
-Test taken from BurntSushi/toml-test
+Test taken from BurntSushi test suite:
+
+    https://github.com/BurntSushi/toml-test
+
+Passing tests are listed in Passing.execopts
```

Comment:
Would you mind calling out the desire to merge Passing and this future together into a single test when the rest pass, either here or in the linked issue?

Reply:
Yes!

Reply:
I mean no, I would not mind.

Reply:
lol, I always have that trouble with answering "would you mind" requests, yet I use them myself anyways XD